pkg/expapi/types.go

```diff
@@ -91,7 +92,27 @@ type HorizontalPodAutoscalerSpec struct {
 	MaxCount int `json:"maxCount"`
```

Comment:
It is possible. We can set minCount to 1 by default and maxCount to Scale.Replicas. WDYT?